### DISCUSSION TITLE
Sunbird Masterclass & Dev Bootcamp for cQube

### DISCUSSION THREAD

**Discussion by DevashishRaghav** [Sunbird-cQube/community#808] [Announcements]
Hey everyone! 📢

Exciting news! 🌟 We have a masterclass coming up that you absolutely can't afford to miss! 🎉

Join us for the "Building for Billions" masterclass on architecting population-scale solutions, delivered by the renowned Dr. Pramod Varma, Ex-Chief Architect of Aadhaar, India Stack, CTO of EkStep & Architect of Sunbird.org! 🏛️

This incredible opportunity is happening at the event, "Lifelong Learning in the Age of AI" on the 21st and 22nd of July 2023. 🗓️ The event will take place at Indraprastha Institute of Information Technology, Delhi (IIIT-Delhi), Okhla Phase-III.

Don't let this chance slip away! 🔥 Register now to secure your spot: https://lu.ma/gbx09ygi 📝

This is a superb opportunity, so be sure to spread the word and invite your friends and colleagues too! 🤝

See you there! 🙌

@dhanush-2397 @htvenkatesh @asifbasha61 @rhwarrier

**Comment by DevashishRaghav** [comment]
### Case collaterals
Dear attendees
Here are all the collaterals for the guided & hands-on case based exercises. Please download the case specific zip as per the discussion in the bootcamp.
[dimensions.zip](https://github.com/Sunbird-cQube/community/files/12122380/dimensions.zip)
[Case.1.zip](https://github.com/Sunbird-cQube/community/files/11855953/Case.1.zip)

**Comment by DevashishRaghav** [comment]
Link to postman collections:

https://drive.google.com/drive/folders/12Xvt6a6Qh1lVoFstD7HCujE7PcJ7vMqv

**Comment by asifbasha61** [comment]
Below are the VM details with public ip and required credentials.
Please use masterclass.pem file provided to you to connect to the VM.

**Instructions to connect to VM:**
Linux or mac:
1. open terminal
2. ssh -i path_to_mastercalss_pem -o ServerAliveInterval=60 ubuntu@<public_ip_of_vm>

Windows:
1. Install GitBash using gitbash using https://git-scm.com/download/win
2. Open GitBash and use below command 
    ssh -i path_to_mastercalss_pem -o ServerAliveInterval=60 ubuntu@<public_ip_of_vm>

**VM Details:**
Table1:
Publc IP: 3.110.180.54
URL:cqube-table1.tibilprojects.com
dashboard creds:
username: table1
Password: Admin@123

Table2:
Public IP: 13.235.252.123
URL: cqube-table2.tibilprojects.com
dashboard creds:
username: table2
Password: Admin@123

Table3:
Public IP: 3.7.54.220
URL: cqube-table3.tibilprojects.com
dashboard creds:
username: table3
Password: Admin@123

Table4:
Public IP: 3.110.109.17
URL: cqube-table4.tibilprojects.com
dashboard creds:
username: table4
Password: Admin@123

Table5:
Public IP: 13.200.6.113
URL: cqube-table5.tibilprojects.com
dashboard creds:
username: table5
Password: Admin@123

Table6:
Public IP: 3.7.74.70
URL: cqube-table6.tibilprojects.com
dashboard creds:
username: table6
Password: Admin@123

Table7:
Public IP: 3.7.54.182
URL: cqube-table7.tibilprojects.com
dashboard creds:
username: table7
Password: Admin@123

Table8:
Public IP: 65.2.116.233
URL: cqube-table8.tibilprojects.com
dashboard creds:
username: table8
Password: Admin@123 

Table9:
Public IP: 13.235.56.17
URL: cqube-table9.tibilprojects.com
dashboard creds: 
username: table9
Password: Admin@123 

Table10:
Public IP: 43.205.91.144
URL: cqube-table10.tibilprojects.com
dashboard creds:
username: table10
Password: Admin@123

    
**Important links and creds:**
1. minio bucket can be accessible using
    a. open broswer
    b. url: <public_ip_of_vm> :9001
        user: minioadmin
        Pswd: minioadmin       
2. Nifi can be accessible using <public_ip_of_vm> :8096   
3. Keycloak can be accessible using 
    url: <url>/auth
    username: admin
    Password: Admin@123    
4. postman collection link
    https://drive.google.com/drive/folders/12Xvt6a6Qh1lVoFstD7HCujE7PcJ7vMqv
